.github/workflows/reuse.yml

```diff
@@ -0,0 +1,18 @@
+# SPDX-FileCopyrightText: 2022 Free Software Foundation Europe e.V. <https://fsfe.org>
+# SPDX-License-Identifier: CC0-1.0
+---
+name: REUSE Compliance Check
+
+on: [push, pull_request]
```

Comment:
Preferred:
```yaml
on:
  push:
    branches:
      - main
  pull_request:
    branches:
      - main
```

Otherwise the workflow is run twice on PRs (as happened here).

Reply:
Is there a benefit from running the linter on `main` at all? Or should we just limit it on  pull requests?

Reply:
It would get us a nice mark in the commit view on the main branch, right?  Otherwise I currently don't know what else.

Reply:
I'm fine with it, as long as the double-execution is solved.